client/src/Utils/Schemas/identification.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
+
+import { passportValidation, passportMaxIdentificationLength , idLength , idBasicValidation} from 'Utils/auxiliaryFunctions/auxiliaryFunctions';
+
+const errorMessage = 'הוכנס תו לא וולידי';
```

Comment:
I would change 'וולידי' to 'חוקי'

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1116/commits/7047524a85644e5143476711afe92d201be6bfdc)